code/modules/reagents/reagent_containers/iv_bag.dm

```diff
@@ -41,21 +41,12 @@
 	..()
 	update_icon(UPDATE_OVERLAYS)
 
-/obj/item/reagent_containers/iv_bag/proc/on_examine(datum/source, mob/examiner, list/examine_list)
-	SIGNAL_HANDLER // COMSIG_PARENT_EXAMINE
-	examine_list += "<span class='notice'>[source.p_they(TRUE)] [source.p_have()] an active IV bag.</span>"
-
```

Comment:
Yeah, any chance we could keep this? It'd save a moment here and there in medbay when juggling blood loss patients.

Reply:
That's not a bad idea, actually. I'll implement that now.